### PR TITLE
buy package deal button on product list and product details, + and - …

### PR DESCRIPTION
**Pull request by Shamzaa**
…button to choose qty of items bought on product details page

### CODE REVIEW

File: frontend/src/app/product-details/product-details.component.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class ProductDetailsComponent implements OnInit, OnDestroy {
   private product: ProductModel;
   private user: UserModel;
   private userAuthEventsSub: Subscription;
+  itemQTY: number;
```
Comment (DeimosA) [inline frontend/src/app/product-details/product-details.component.ts]: private

File: frontend/src/app/product-details/product-details.component.html [hunk 2]
```diff
@@ -36,8 +36,19 @@ <h3 *ngIf="product.on_sale === 'PRICE_MOD'">
             <h3 *ngIf="product.on_sale !== 'PRICE_MOD'"> kr {{product.price.toFixed(2)}} </h3>
           </div>
 
+          <div class="row">
+            <button (click)="iterateDown()" type="button" class="btn-floating waves-effect">
+              <i class="material-icons">remove</i>
+            </button>
+            x{{ itemQTY }}
+            <button (click)="itemQTY = itemQTY + 1" type="button" class="btn-floating waves-effect">
```
Comment (DeimosA) [inline frontend/src/app/product-details/product-details.component.html]: +=

File: frontend/src/app/product-details/product-details.component.ts [hunk 3]
```diff
@@ -35,15 +36,20 @@ export class ProductDetailsComponent implements OnInit, OnDestroy {
...
   }
 
-  addToCart(id){
+  addToCart(id, qty){
     // adding item ID to cart
-    this.shoppingCart.addItem(id);
+    this.shoppingCart.addItem(id, qty);
+  }
+
+  iterateDown(){
```
Comment (DeimosA) [inline frontend/src/app/product-details/product-details.component.ts]: decrementQuantity

**Review by DeimosA — CHANGES_REQUESTED** [review]

File: frontend/src/app/product-details/product-details.component.html (hunk 2 above)
Comment (Shamzaa) [reply frontend/src/app/product-details/product-details.component.html]: += didn't work

**Comment by Shamzaa** [comment]
closes #98

**Review by DeimosA — APPROVED** [review]

**Pull request merged**